webapp/src/components/testing/TestSteps.vue

```diff
@@ -228,59 +226,59 @@ const visitedSamples = computed<number>(() => {
   }
 });
 
-const confusionMatrix = computed<{ [key: string]: { [key: string]: number } } | undefined>(() => {
-  if (tested.value === undefined) return undefined;
+const confusionMatrix = computed<Map<string, Map<string, number>> | undefined>(
```

Comment:
<img width="732" height="268" alt="confusion_matrix" src="https://github.com/user-attachments/assets/baf0b854-020e-43d6-9b38-5646c2b1f345" />
The confusion matrix is not being displayed correctly